java/rocksjni/compaction_filter_factory_jnicallback.h

```diff
@@ -26,7 +26,7 @@ class CompactionFilterFactoryJniCallback : public JniCallback, public Compaction
     virtual const char* Name() const;
 
  private:
-    std::unique_ptr<const char[]> m_name;
+    std::unique_ptr<char[]> m_name;
```

Comment:
Why?

Reply:
because is inconsistent with other parts where you use it